CRM/Contact/Form/DedupeFind.php

```diff
@@ -51,6 +51,7 @@ public function buildQuickForm() {
     $groupList = array('' => ts('- All Contacts -')) + CRM_Core_PseudoConstant::nestedGroup();
 
     $this->add('select', 'group_id', ts('Select Group'), $groupList, FALSE, array('class' => 'crm-select2 huge'));
+    $this->add('text', 'limit', ts('No of contacts to find matches for '));
```

Comment:
the main reason for it is to protect people from crashing the db inadvertently - which is what no limit does on a large group or site - that's the problem I'm trying find a solution for